### ISSUE TITLE
Update Error text when required field is missing

### ISSUE THREAD

**Issue by ClaireValdivia** [usdigitalresponse/usdr-gost#3053]
### Why is this issue important?

the error message is not correct when a required field is missing

### Current State

When missing a required field, error message is "Error in field [field name]-Input should be a valid string" 

### Expected State

The message should be "Value is required for [field name]"

### Implementation Plan

The following functions need to be added...

**Comment by vshia** [comment]
@ClaireValdivia  @as1729 ? Is the `project_use_code` the same as the EC code? 

Also, what does an example good and bad worksheet look like?

**Comment by ClaireValdivia** [comment]
started thread in slack for discussion, but popping in the link to test files here as well! https://drive.google.com/drive/folders/1lPoniSxq_GPQfiP723QBhNpR3HR8dXeb

**Comment by as1729** [comment]
@vshia that is correct - the field `Project Use Code` with name in code as `project_use_code` is what we are referring to as EC Code in this ticket.

![Image](https://github.com/usdigitalresponse/usdr-gost/assets/6497366/038cc13e-ecbc-4650-b63a-e9ef6018fa6f)


@ClaireValdivia I'm wondering whether this is a good opportunity to update the name to `ec_code` in the input template?

**Comment by ClaireValdivia** [comment]
oooh yes great call. I'll add this to my list of template updates.

**Comment by vshia** [comment]
Here's the first PR to go in: https://github.com/usdigitalresponse/cpf-reporter/pull/276
I'm going to add another quick change to ensure that inputs that are just spaces (i.e. ` `) do not pass validation.
